src/OpenLoco/Widget.cpp

```diff
@@ -334,16 +333,16 @@ namespace OpenLoco::Ui
             window->y + top + 40,
             shade);
 
-        draw_resize_handle(context, window, widget, colour);
+        draw_resize_handle(context, window, this, colour);
     }
 
     void Widget::draw_3(Gfx::Context* context, const Window* window, uint16_t flags, uint8_t colour, bool enabled, bool disabled, bool activated)
     {
-        int16_t top, left, bottom, right;
-        top = window->y + top;
-        left = window->x + left;
-        right = window->x + right;
-        bottom = window->y + bottom;
+        int16_t top_rect, left_rect, bottom_rect, right_rect;
+        top_rect = window->y + top;
```

Comment:
one of the other functions in this file uses t, l, r, b for the variables. I think in this context there is enough information for that to be sufficient for the variable names.